aws-android-sdk-s3/src/test/java/com/amazonaws/mobileconnectors/s3/transferutility/UploadInputStreamTest.java

```diff
@@ -112,6 +112,7 @@ public void testUploadWithAllParametersSet() throws IOException {
                                                               .build());
         verify(transferUtility, times(1)).upload(eq(bucket), eq(key), any(File.class),
                 eq(metadata), eq(cannedAcl), eq(listener));
+        transferUtility.getDbUtil().closeDB();
```

Comment:
`tearDown()` method is called after each test case, so this will be called later anyways. So let's just remove this line.